app/components/channel_icon/dm_avatar/dm_avatar.tsx

```diff
@@ -24,11 +25,24 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     statusInfo: {
         backgroundColor: theme.centerChannelBg,
     },
+    icon: {
+        color: changeOpacity(theme.sidebarText, 0.4),
+    },
 }));
 
 const DmAvatar = ({author, isInfo}: Props) => {
     const theme = useTheme();
     const style = getStyleSheet(theme);
+
+    if (author?.deleteAt && author.deleteAt > 0) {
+        return (
+            <CompassIcon
+                name='archive-outline'
+                style={[style.icon, {fontSize: 24, left: 1}]}
```

Comment:
Any reason why font size and left are not part of the icon style?

Reply:
Also, the size can be directly passed to the `size` prop for CompassIcon (which is a bit more readable in my opinion).